runelite-api/src/main/java/net/runelite/api/Experience.java

```diff
@@ -26,9 +26,7 @@
 
 public class Experience
 {
-	/**
-	 * Maximum level under 200m xp
-	 */
+	//Maximum level under 200m xp
```

Comment:
Why the removal of the Javadoc comment here?

Reply:
I hadn't realized that was a javadoc comment, so I figured I'd squish down the multiline comment to a single line comment. I'll change it back